src/wabt.cpp

```diff
@@ -57,12 +57,15 @@ class WabtEthereumInterface : public EthereumInterface {
     m_wasmMemory = _wasmMemory;
   }
 
+  std::exception_ptr my_exception_ptr;
```

Comment:
Can you please not go with `my_exception_ptr`, rather something like `last_exception_ptr` ?

Reply:
Ok.